adr/ADR-20.md

```diff
@@ -388,13 +395,20 @@ Watch(opts ...WatchOpt) -> ObjectWatcher
 **List**
 
 List will list all the objects in this store.
- 
-* Should not include deleted objects.
+* Do not include deleted objects, except with optional convenience methods.
 
 ```
+List() -> List or array of ObjectInfo
+```
+
+Optional/Convenience examples:
+
+```
+List(includingDeleted bool) -> List or array of ObjectInfo
```

Comment:
don't put this one, when you suggest the other, because as soon as we want to filter the list of objects to those that match a particular name or size or date, you'll have to add API which the options below allow you to add without breaking or adding additional versions of list.

Reply:
removed